public/components/common/util/markdown.tsx

```diff
@@ -0,0 +1,33 @@
+/*
+ * Wazuh app - React component that renders markdown
+ * Copyright (C) 2015-2021 Wazuh, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * Find more information about this on the LICENSE file.
+ */
+import React from 'react';
+import MarkdownIt from 'markdown-it';
+import classnames from 'classnames';
+
+const md = new MarkdownIt({
+  html: true,
+  linkify: true,
+  breaks: true,
+  typographer: true
+});
+
+interface MarkdownProps{
+  markdown: string
+  className?: string
+};
+
+export const Markdown = ({markdown, className = ''}: MarkdownProps) => (
+  <div
+    className={classnames('wz-markdown-margin', 'wz-markdown-wapper', className)}
+    dangerouslySetInnerHTML={{__html: md.render(markdown)}}>
```

Comment:
we need to avoid using `dangerouslySetInnerHTML`, but if it is totally necessary we should use a sanitizer to avoid XSS
i.e. [DOMPurify](https://github.com/cure53/DOMPurify)

```js
import { sanitize } from 'dompurify';
...
 <div
    className={classnames('wz-markdown-margin', 'wz-markdown-wapper', className)}
    dangerouslySetInnerHTML={{__html: sanitize(md.render(markdown))}}>
  </div>
```

Reply:
maybe is a good idea to leave a justification like Kibana does
![image](https://user-images.githubusercontent.com/26828184/122060167-b8f03980-cdc3-11eb-9211-9af41168335e.png)

Reply:
The content of the `markdown` comes from the Wazuh API response. Do we really need to add that dependency to the app to this use case?

Reply:
Added comment with the justification to using `dangerouslySetInnerHTML`.